front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/ObjectType.java

```diff
@@ -28,6 +29,7 @@
   PROJECT(Project.class, "projects", "project-metadata.json"),
   PIPELINE(Pipeline.class, "pipelines", "pipeline-metadata.json"),
   STRATEGY(Pipeline.class, "pipeline-strategies", "pipeline-strategy-metadata.json"),
+  PIPELINE_TEMPLATE(PipelineTemplate.class, "pipelineTemplates", "pipeline-template-metadata.json"),
```

Comment:
maybe `pipeline-templates` vs `pipelineTemplates` to be consistent with the naming convention of other types.

![image](https://cloud.githubusercontent.com/assets/388652/25419647/c31f4986-2a08-11e7-8f13-1b464b9f315c.png)